src/GraphQL/Execution/Nodes/ExecutionNode.cs

```diff
@@ -50,7 +50,7 @@ public abstract class ExecutionNode
             {
                 if (IndexInParentNode.HasValue)
                 {
-                    return ((ListGraphType)Parent!.GraphType).ResolvedType;
+                    return ((ListGraphType?)Parent!.GraphType)?.ResolvedType;
```

Comment:
😨

Reply:
Could be a possible bug I found. Can’t remember.